### ISSUE TITLE
Server error 500 when reset password

### ISSUE THREAD

**Issue by fadyscube** [ykdojo/csqa#37]
I want to change my account's password for a reason, so I've tried the forgot password option and when I've entered my email and submit the form, I got a 500 error from the server. I want to change my csqa account's password.

**Comment by ykdojo** [comment]
Hey, thanks for letting me know about this. I'll get back to you when I fix
it!

**Comment by ykdojo** [comment]
Actually, I'm now thinking of shutting down the website and redirecting people to my subreddit. Thoughts? @fadyscube

**Comment by ykdojo** [comment]
(It's just that I haven't been able to spend much dev time on this :()

**Comment by fadyscube** [comment]
As you like YK

**Comment by Rachit74** [comment]
> Actually, I'm now thinking of shutting down the website and redirecting people to my subreddit. Thoughts? @fadyscube

yes, you made a really cool website but there are a lot of issues, a lot of people always spam in the comments many of the actual questions are unanswered due to the spam questions, you will be able to manage subreddit easily and you can make some bot or moderators to manage. you can still keep up the source but i think taking down the actual website would be the best decision. (I'm not trying to force you to shut down the site I really love the site but the circumstances are not so in the favor of keeping the site up.)
